gems/holiday_jp/0.8/holiday_jp.rbs

```diff
@@ -0,0 +1,43 @@
+module HolidayJp
+  def self.between: (untyped start, untyped last) -> (::Array[Holiday])
+
+  def self.holiday?: (untyped date) -> bool
+
+  def self.holidays: () -> untyped
+end
+
+module HolidayJp
+  class Holiday
+    WDAY_NAMES: ::Array[::String]
+
+    EN_HOLIDAY_NAMES: ::Hash[::String, ::String]
+
+    attr_reader date: untyped
+
+    attr_reader name: ::String
+
+    def initialize: (untyped date, ::String name) -> void
+
+    def name_en: () -> ::String
+
+    def wday_name: () -> ::String
+
+    alias week wday_name
+  end
+end
+
+module HolidayJp
+  class Holidays
+    attr_accessor holidays: untyped
```

Comment:
As far as I can see, `Holidays#holidays` seems to [be initialized as an instance of `Hash`](https://github.com/holiday-jp/holiday_jp-ruby/blob/ef8f0370b1119dca348f8dc19ec0a7154f500d26/lib/holiday_jp/holidays.rb#L8). Can we declare it as `Hash[untyped, untyped]`?

```suggestion
    attr_accessor holidays: Hash[untyped, untyped]
```

Reply:
@yykamei  
You are right!. I will correct it as such.
Thank you for comment.